### ISSUE TITLE
Move from Dockerize to Waiting on KV Store

### ISSUE THREAD

**Issue by myhndl** [polyswarm/polyswarmd#59]
**Is your feature request related to a problem? Please describe.**
Dockerize leaves a bit to be desired. We're moving to a distributed key-value (KV) store (Consul), so we should coordinate handoffs via this store.

**Describe the solution you'd like**
Rather than waiting for contracts to be deployed via Dockerize or file touches, block on the availability of a Consul key. When available, this key points to (its value is) necessary chain data.

**Comment by scsich** [comment]
Closing in favor of #54 which is already on its way.
